pixie/io.pxi

```diff
@@ -172,6 +172,20 @@
        (write-byte fp chr)
        nil))))
 
+(defn stream-output-rf [output-stream]
+  (let [fp (buffered-output-stream output-stream
+                                   DEFAULT-BUFFER-SIZE)]
+    (fn ([] 0)
+      ([_] (dispose! fp))
+      ([_ chr]
+       (assert (integer? chr))
+       (write-byte fp chr)
+       nil))))
+
+(defn write-stream [output-stream val]
```

Comment:
This should be run through the UTF8 logic in pixie.streams.utf8. But I'm fine with that being in a different PR. I need to abstract the UTF8 stuff a bit anyways.